recipe/meta.yaml

```
@@ -13,17 +13,22 @@ source:
 
 build:
   skip: true  # [win]
-  number: 0
+  number: 1
   script: {{ PYTHON }} -m pip install . --no-deps -vv
 
 requirements:
   build:
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - numpy                                  # [build_platform != target_platform]
     - {{ compiler('c') }}
   host:
     - python
+    - numpy
     - pip
   run:
     - python
+    - numpy
```

Should this be

```suggestion
    - {{ pin_compatible('numpy') }}
```

based on: https://conda-forge.org/docs/maintainer/knowledge_base.html#building-against-numpy